tests/lib/rules/indent.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @fileoverview Check indentation at the beginning and end of a function call
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require('../../../lib/rules/indent'),
+    formatCode = require('../../code-helper'),
+    RuleTester = require('eslint').RuleTester,
+    ruleTester = new RuleTester(),
+    invalidExample, validExample;
+
+invalidExample = formatCode(
+   'function test() {',
+   '   var a = 1,',
+   '      b = 2;',
+   '',
+   '   return a + b;',
+   '}'
+);
+
+validExample = formatCode(
+   'function test() {',
+   '   var a = 1,',
+   '       b = 2;',
+   '',
+   '   return a + b;',
+   '}'
+);
```

Comment:
Please add more invalid and valid examples, including our fluent-chaining examples. This rule needs to check *all* indentation standards, not just for `var` declarations. So it should make sure that statements in a function are indented one from a function, etc. Think of complex nesting situations and try them here with valid and invalid examples and make these tests more useful and not so surface level. Thanks.